### DISCUSSION TITLE
Mi práctica HTML - Sugar Bakery

### DISCUSSION THREAD

**Discussion by hackit92** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#184] [Show and tell]
Que tal compañeros, quisiera compartir la dirección de mi página de práctica, aún me falta desarrollar el formulario y corregir unos problemitas con la entrega de imagenes por https, saludos!

Edit: Realicé una actualización corrigiendo el tema de las imágenes, añadí el formulario y el acceso al dashboard mediante un login (solo frontend) Gracias a todos por sus buenos comentarios =)

https://frontend.wintecservices.com/

**Comment by ArmandoRiveraT** [comment]
Wow se ve profesional, ya tenias conocimiento de html y css?

**Reply by hackit92** [reply]
Si, ya conozco algunas cosas de frontend, de hecho mi idea de ingresar a este bootcamp fue principalmente por las tecnologías más avanzadas de frontend y backend porque de eso se relativamente poco. Muchas gracias por tu opinión

**Reply by vicogarcia16** [reply]
No sabia que la podias hacer en bootstrap

**Reply by hackit92** [reply]
Pues no se, no vi ninguna restricción y de ahí me agarré para intentar realizar algo un poco más elaborado que solo la maqueta HTML, sin embargo no me atrevo a decir que si se puede jeje

**Reply by patricia2454** [reply]
Hola @hackit92 al ver tu página me senti desanimada, ya que la mía quedo demás de sencilla, también la subí pero, nada que ver a tu trabajo... Muchas felicidades, se ve super padre; me reconforta que ya tenias nociones previas; toca esforzarme para que se vea igual de linda que la tuya!

**Reply by hackit92** [reply]
Que tal @patricia2454 te agradezco mucho por tu comentario hacia mi práctica, pero quisiera que de ninguna forma sea motivo de desánimo, llevo ya un tiempo orientando mi carrera hacia la web y lo que la hace tan vistosa es que yo comencé como diseñador, por lo que en estética tengo ventaja, pero te aseguro que con algo de empeño y dedicación podrás realizar esto y más =) Si quieres puedo recomendarte algunos cursos para que puedas aprender más rápido a dominar estas tecnologías, excelente noche!

**Reply by patricia2454** [reply]
Claro que sí, agradezco tu atención y respuesta! quedo en espera de la recomendación :) @hackit92

**Reply by hackit92** [reply]
@patricia2454 hay muchisimo contenido enfocado en web aquí y es gratis =) https://www.faztweb.com/

**Reply by JosePabloGrr** [reply]
Felicidades al compañero por compartir su trabajo, Para responder la pregunta de @vicogarcia16 ... **el MC dijo en el ultimo Stream que no se limiten, si ya tienen conocimiento pueden utilizar bootstrap u otro frame** (ya que alguien pregunto si podia inplementar funcionamiento, aunque se dijo que no es indispensable de momento), Se trata de poner en practica lo que conocen y experimentar en caso de nuevas tecnologias :) pero que estan enseñando lo basico para todos ir a la par 🙌🏻

**Comment by nataliaramosj** [comment]
esta hermosa tu pagina :)  yo apenas estoy aprendiendo ..  mi pagina se ve muy básica pero bueno , poco a poco .

**Reply by hackit92** [reply]
Muchas gracias! :D Si, es poco a poco, me encantaria ver la tuya

**Comment by LuceroLuciano** [comment]
Explorer te quedo increíble tu página, 100/100!
Me imagino que utilizaste alguna otra librería o framework para la parte de diseño, o solo boostrap y css?
Te quedo super bonita.

**Reply by hackit92** [reply]
Muchas gracias :D Efectivamente esta hecha con bootstrap y algo de CSS personalizado para algunas clases adicionales

**Comment by OscarIslasH** [comment]
Te quedo extremadamente buena ! Te salio muy bien !

Donde conseguiste las imagenes y los iconos?

**Reply by hackit92** [reply]
Muchas gracias! Las imagenes son de pexels y los iconos de la libreria de fontawesome :)

**Comment by martinmurillo2020** [comment]
> Que tal compañeros, quisiera compartir la dirección de mi página de práctica, aún me falta desarrollar el formulario y corregir unos problemitas con la entrega de imagenes por https, saludos!
> 
> https://frontend.wintecservices.com/

No la puedo ver, en mi navegador no muestra nada después de la barra

**Reply by hackit92** [reply]
Que extraño, ojalá puedas ayudarme con el log de la consola y una captura de pantalla, gracias por reportarlo

**Comment by JuanSalvador73** [comment]
Muy buen trabajo, felicidades. :D

**Reply by hackit92** [reply]
Gracias!

**Comment by jmlimas** [comment]
[hackit92] Donde subiste  tu pagina es   gratis..?

**Reply by hackit92** [reply]
Hola! Tengo mis propios servidores, pero te recomendaría echarle un ojo a AWS y firebase, tienen excelentes opciones sin costo adicional

**Comment by emonreal9604** [comment]
se ve genial, muy profesional.

**Reply by hackit92** [reply]
Muchas gracias!

**Comment by sergiolopezloya** [comment]
Wow, le echaste muchas ganas, toda una página web terminada
